src/style.css

```diff
@@ -0,0 +1,52 @@
+.gadget {
+  position: relative;
+  font-family: "Roboto", sans-serif;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-weight: 500;
+  font-size: 12px;
+  margin-left: 20px;
+  line-height: 60px;
+  color: #000;
+  display: inline-block;
+}
+
+.box {
+  display: flex;
+  flex-direction: row;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.is-active {
+  margin-left: 0;
```

Comment:
if we add this class to another element of the list?

Reply:
![image](https://user-images.githubusercontent.com/44294945/64178744-b761b580-ce6a-11e9-89ee-fdafa7b97fc9.png)